internal/app/app.go

```diff
@@ -138,7 +141,7 @@ func NewCalendar(ctx context.Context, logger log.Logger) calendar.Calendar {
 			calendarID    = config.Env.GoogleCalendarID
 		)
 		if !googlecalendar.ValidateParameters(calendarID, calendarToken) {
-			logger.Error(
+			logger.Warn(
```

Comment:
I would actually keep this one as Error, since there is already an option to set calendar as "none", so if "google_calendar" is actually selected, the parameters should be configured correctly.